### DISCUSSION TITLE
Cant get my buttons the same size

### DISCUSSION THREAD

**Discussion by bprosman** [twbs/bootstrap#37331] [Q&A] [answered]
Hi,
I am using the bootstrap 4 "buttons" but cant get them to the same size.
As the screenshot shows , the 2000 buttons are larger than the 199x buttons.
Anyone has an idea ?
![Button-diff-sizes](https://user-images.githubusercontent.com/46261397/196357588-14db9fd4-008f-4ce5-989b-f6baaabe64e5.JPG)

My code
![Button-Code](https://user-images.githubusercontent.com/46261397/196357937-92f53081-219d-4f0f-9d1b-7cab2082b5d3.JPG)

**Comment by mdo** [comment, marked as answer]
As in the same width? That's because individual letters in the font aren't the same width. You'd need to apply a fixed `width` to make then all equal, or use some other CSS to make equal-width children.

**Reply by bprosman** [reply]
The font difference is what I noticed as well. As I am a fully PHP/CSS amateur/beginner, could you give me a hint what I could add to the CSS ? Or change the font ?

**Reply by mdo** [reply]
You could add `width` to any of them (you'd have to check what size to include). Or you could add `.text-monospace` to use a monospace font that has more consistent letter sizes.

Personally I don't think it's worth doing either of those, subtle differences across browsers and fonts are to be expected :). But, totally your call!

**Reply by bprosman** [reply]
Let me try, anyway thanks for reacting to my question.
